BCH-Wunder-Project-team-4/Wunder.io
# About us

## Link to ticket:

[Ticket](https://trello.com/c/TUYKex6A)

## Link to feature environment:

[Start your journey here](http://localhost:3000/en/about-wunder)

## Changes proposed in this PR:

- Adds content for About Wunder page, and three subpages (Culture, For Media, Wunder Story) and Privacy Policy page
- Several new paragraphs (Story Block, Text and Image, Contact Details, Office Details)
- Styling adjustments here and there
- Translated Main Menu links to Finnish
- There are Finnish versions for the pages I created but no translations added (Just the title)
- Enables Telephone Field type
- New content type About Us Subpage

## How to test:

**Create a backup before starting** `lando db-export`  **(Just in case)**

`git fetch origin aboutUs`
`git checkout aboutUs`
`lando drush cim`
`lando db-import DBlatestAboutUs.sql.gz`

Check out the new pages (About and Privacy policy) that content and pictures are there.

## Screenshots

![Screenshot 2023-12-20 at 2 54 30](https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/122397061/e0594c2f-a5fc-4e59-b39f-c9fe823b7cd2)
![Screenshot 2023-12-20 at 2 27 02](https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/122397061/26b6d006-867f-4074-81a5-b6994f45ac70)

## Best practices:

<details>
<summary><h3>Accessibility:</h3></summary>
<p>
This project must support WCAG accessibility level AA <em>(edit this according to the requirements of your project)</em>. To ensure this standard is met, remember to:

- Perform automated checks using a tool such as Wave or SiteImprove.
- Test keyboard navigation: are all parts of the UI navigable using only the keyboard? Is the tab order logical? Can popups, menus etc be dismissed with the escape key?
- Test responsiveness, scaling and text reflow.
- Make sure no accessibility issues exist on either desktop or mobile views.
- If you have time, test with a screen reader such as VoiceOver (macOS), NVDA (Windows), or Orca (Linux).

Use the [Accessibility Testing Cheat Sheet](https://intra.wunder.io/info/accessibility-group/accessibility-testing-cheat-sheet) for information on how to run these tests.
</p>
</details>
